lib/graphql/errors.rb

```diff
@@ -49,6 +51,18 @@ def rescue_from(*classes, &block)
 
     private
 
+    def wrap_proc(object, arguments, context, old_resolve_proc)
```

Comment:
😻 

we might want to rename `old_resolve_proc` arg to something else (`proc`, `old_proc`, etc.)